File: algorithms/warmup/sherlock_beast.py
def pswd(n):
    if n % 3 == 0:
        return "5" * n

    pswd = ""
    while (n % 3 != 0):
        pswd = pswd + "33333"
        n -= 5
        if n < 0:
            return -1
    pswd = "5" * n + pswd
    return pswd

for i in range(0, input()):
    print pswd(input())

File: algorithms/warmup/sherlock_beast.cpp
#include <iostream>
#include <string>

std::string password(int n);

int main() {
    int t;
    std::cin >> t;
    int n;
    for (int i = 0; i < t; ++i) {
        std::cin >> n;
        std::cout << password(n) << std::endl;
    }
    return 0;
}

std::string password(int n) {
    std::string pswd = "";
    
}
